docroot/modules/custom/cs_generated_content/generated_content/media/civictheme_document.inc

```diff
@@ -11,40 +11,40 @@ use Drupal\generated_content\Helpers\GeneratedContentAssetGenerator;
 use Drupal\media\Entity\Media;
 
 /**
- * Implements hook_cs_generated_content_create_ENTITY_TYPE_BUNDLE_weight().
+ * Implements hook_generated_content_create_ENTITY_TYPE_BUNDLE_weight().
  */
 function cs_generated_content_generated_content_create_media_civictheme_document_weight() {
   return 10;
 }
 
 /**
- * Implements hook_cs_generated_content_create_ENTITY_TYPE_BUNDLE_tracking().
+ * Implements hook_generated_content_create_ENTITY_TYPE_BUNDLE_tracking().
  */
 function cs_generated_content_generated_content_create_media_civictheme_document_tracking() {
   return TRUE;
 }
 
 /**
- * Implements hook_cs_generated_content_create_ENTITY_TYPE_BUNDLE().
+ * Implements hook_generated_content_create_ENTITY_TYPE_BUNDLE().
  */
 function cs_generated_content_generated_content_create_media_civictheme_document() {
   /** @var \Drupal\cs_generated_content\CsGeneratedContentHelper $helper */
   $helper = CsGeneratedContentHelper::getInstance();
 
-  $total_files_count = 30;
-
-  $filename = sprintf('Demo document file %s', $helper::staticString());
-  $file = $helper::createFile(GeneratedContentAssetGenerator::ASSET_TYPE_PDF, [
-    'filename' => $filename,
-  ], FALSE);
+  $total_media_count = 30;
 
   $entities = [];
-  for ($i = 0; $i < $total_files_count; $i++) {
+  for ($i = 0; $i < $total_media_count; $i++) {
     $variation = $helper::variationRandomValue([
       'status' => NULL,
     ]);
 
-    $name = sprintf('Demo Document media %s %s', $i + 1, $helper::staticSentence(2));
+    $file = $helper->staticFile(GeneratedContentAssetGenerator::ASSET_TYPE_PDF);
```

Comment:
here and below - instead of creating file entities in each media, we are now using helpers to retrieve previously created File entities